images/client/Makefile

```diff
@@ -1,11 +1,19 @@
+CONTAINER_CMD ?=
```

Comment:
This is one reason a single makefile may be better. No need to repeat yourself like this. :-)

Reply:
I agree. exactly my thought when I did the patches... :grin: 
Room for more improvements...